docs/cli/cmdline/cloud-drift-show.md

```diff
@@ -0,0 +1,22 @@
+---
+title: terramate cloud drift show - Command
+description: With the terramate cloud drift show command you can view the drift that has occurred on a stack
+
+prev:
+  text: 'Clone'
+  link: '/cli/cmdline/clone'
+
+next:
+  text: 'Cloud Info'
+  link: '/cli/cmdline/cloud-info'
+---
+
+# Cloud Drift Show
+
+**Note:** This is an experimental command that is likely subject to change in the future.
+
+The `cloud drift show` command shows any drift that has occurred in the current stack in the working directory (or use `-C` to point to the directory). It requires a working Terramate Cloud account.
```

Comment:
What about saying it needs to be logged in and maybe link to the `./cloud-login.md`?

Reply:
Fixed, though we're going to be making a lot of changes over the next month and will probably have a unified guide we'll link to for setting up local environment